examples/on-conflict-clause/on_conflict_clause.bal

```diff
@@ -7,32 +7,53 @@ type Student record {|
 |};
 
 public function main() {
-    error onConflictError = error("Key Conflict", message = "record with same key exists.");
-
     table<Student> students = table [
-            {id: 1, name: "John", score: 100},
-            {id: 2, name: "Jane", score: 150},
-            {id: 1, name: "John", score: 200}
+            {id: 1, name: "Admin", score: 100},
+            {id: 2, name: "John", score: 150},
+            {id: 1, name: "Admin", score: 200}
         ];
 
-    // The result of the following will be an error since the key `John` is duplicated.
+    // The result of the following will be an error since the key `Admin` is duplicated
+    // and `isSafeReplace()` function returns an error for the name `Admin`.
     map<int>|error studentScores = map from var {name, score} in students
                                    select [name, score]
-                                   on conflict onConflictError;
+                                   on conflict isSafeReplace(name);
 
     io:println(studentScores);
 
-    // The value `100` of the key `John` will be replaced by the second occurrence value `200`.
-    map<int> lastRoundScore = map from var student in students
-                              select [student.name, student.score]
-                              on conflict ();
-
-    io:println(lastRoundScore);
-
-    // The result of the following will be an error since the key `1` is duplicated.
+    // The result of the following will be an error since the key `1` is duplicated
```

Comment:
key 1?

Reply:
Yes in this case the `id` is the key, not the `name`, and the value `1` for the `id` is duplicated